app/assets/stylesheets/wave-spinner.css.scss

```diff
@@ -0,0 +1,31 @@
+$spinkit-spinner-margin: 20px auto !default;
+$spinkit-spinner-color: #333 !default;
+
+.sk-wave {
+  $rectCount: 5;
+  $animationDuration: 1.2s;
+  $delayRange: 0.4s;
+
+  margin: $spinkit-spinner-margin;
+  width: 50px;
+  height: 40px;
+  text-align: center;
+  font-size: 10px;
+
+  .sk-rect {
+    background-color: $spinkit-spinner-color;
+    height: 100%;
+    width: 6px;
+    display: inline-block;
+    animation: sk-waveStretchDelay $animationDuration infinite ease-in-out;
+  }
+
+  @for $i from 1 through $rectCount {
+    .sk-rect#{$i} { animation-delay: - $animationDuration + $delayRange / ($rectCount - 1) * ($i - 1); }
```

Comment:
Name of variable `animationDuration` should be written in all lowercase letters with hyphens instead of underscores<br>Name of variable `delayRange` should be written in all lowercase letters with hyphens instead of underscores<br>Name of variable `rectCount` should be written in all lowercase letters with hyphens instead of underscores